WcaOnRails/db/migrate/20180709220826_add_created_at_column_to_competitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedAtColumnToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :created_at, :datetime
```

Comment:
If at some point you need to add both timestamps instead of just `created_at` you can use `add_timestamps :Competitions`.

Also do we want to allow for null values here?